ansible/roles/db/molecule/default/tests/test_default.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
+
+# check if MongoDB is enabled and running
```

Comment:
It is better to not write "obvious" comments (as the rule: add comments only where they are strictly necessary)